src/client/search/SearchActions.spec.js

```diff
@@ -0,0 +1,52 @@
+import { should } from 'chai';
+import * as actions from './SearchActions';
+
+describe('search actions', () => {
+  it('should create an action to add a hashtag', () => {
+    const id = 0;
+    const query = '#Footy';
+
+    const expectedQuery = 'Footy';
+    const expectedAction = {
+      type: 'ADD_SEARCH_TERM',
+      id,
```

Comment:
This is es6 syntax turns into:
const expectedAction = {
  type: 'ADD_SEARCH_TERM',
  id: id //the id const declared at the top, just shorthand to write id
  ...
};